index.html

```diff
@@ -45,6 +45,238 @@ <h4>i am a software developer</h4>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
```

Comment:
Please make individual cards as child elements of the work section as instructed by the previous reviewer. By making them as child elements, I mean wrapping all project cards into the work section. I also recommend, using `<article>` or `<div>` for each card, since a `section` should never contain another `section`.